media_types/_util.ts

```diff
@@ -83,14 +84,15 @@ export function decode2331Encoding(v: string): string | undefined {
   if (sv.length !== 3) {
     return undefined;
   }
-  const charset = sv[0].toLowerCase();
+  const [sv0, , sv2] = sv as [string, string, string];
```

Comment:
Since we check the string length is 3, we know that indices 0,1,2 will always be string.